### PR TITLE
Harmonize FLOSS blocks docs

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
Overall PR: https://github.com/up42/snap-polarimetric/pull/9
Tracked by https://geoinformationstore.atlassian.net/browse/UP-3132

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,177 @@
...
 ```bash
-make validate
+make e2e
 ```
 
-Next build the docker image
+
+## Pushing the block to the UP42 platform
+
+First login to he UP42 docker registry. `me@example.com` needs to be replaced by your **UP42 username**,
```
Comment (nekhtiari) [inline README.md]: :D

**Comment by nekhtiari** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,87 +1,177 @@
...
 
 ```bash
 make install
 ```
 
-You can run all unit tests with:
+## Testing the block locally
+
+Before uploading the block to the UP42 platform, we encourage you to run the following local tests and validations to
+ensure that the block works as expected, conforms to the UP42 specifications and can could successfully applied in an
```
Comment (chrieke) [inline README.md]:
```suggestion
ensure that the block works as expected, conforms to the UP42 specifications and could be successfully applied in an
```

File: README.md [hunk 3]
```diff
@@ -1,87 +1,177 @@
...
 
 This is an example data block based on NASA's [Global Imagery Browse Services (GIBS)](https://earthdata.nasa.gov/eosdis/science-system-description/eosdis-components/gibs) providing access to three-band
-MODIS imagery.
+MODIS imagery. This block can also provide other MODIS derived data such as the Normalized Difference Vegetation Index (NDVI) (rolling 8-day average) product or additional MODIS bands such as Corrected Reflectance for Bands 7-2-1.
 
-## Block description
-* Block type: data   
-* Output type: AOIClipped (geo-referenced [GeoTIFF](https://en.wikipedia.org/wiki/GeoTIFF))
-* Provider: [UP42](https://up42.com)
+t functions as an example data block. The block functionality and performed
```
Comment (chrieke) [inline README.md]: something missing?

Comment (rodrigoalmeida94) [reply README.md]: Good point, corrected the sentence.

**Pull request merged**